app.js

```diff
@@ -1,88 +1,758 @@
+// 6. true && !(false || false) Prediction: true Result: true
+// 7. !true && !(false || false) Prediction: false Result: false
+// 8. 'cat' + 'dog' Prediction: catdog Result: 'catdog'
+// 9. 2 + 'pizza' Prediction: 2pizza Result: '2pizza'
+// 10. 2.5 * 'fish' Prediction: undefined Result: NaN *****
+// 11. 5 >= 11 Prediction: false Result: false
+// 12. 5 === 5.0 Prediction: false Result: true *****
+// 13. 7 !== 7.1 Prediction: true Result: true
+// 14. 5 + 5 > 8 Prediction: true Result: true
+// 15. 6 + 6 !== 12 Prediction: false Result: false
+// 16. 2 > 1 || false Prediction: true Result: true
+// 17. 'true' === true Prediction: true Result: false *****
+// 18. 10 % 2 === 0 Prediction: true Result: true
+// 19. 21 % 2 === 0 Prediction: false Result: false
+// 20. 21 % 2 !== 0 Prediction: true Result: true
+// 21. 21 % 2 === 1 Prediction: true Result: true
+// 22. 12 % 3 === 0 Prediction: true Result: true
+// 23. 9 % 3 === 0 Prediction: true Result: true
+// 24. 14 % 3 === 0 Prediction: false Result: false
+// 25. 'new york'[0] Prediction: n Result: n
+// 26. 'new york'[1] Prediction: e Result: e
+// 27. 'san francisco'[2 * 3] Prediction: a Result: a
+// 28. 'engineering'[3].toUpperCase() Prediction: I Result: 'I'
+// 29. 'engineering'.indexOf('G') Prediction: undefined Result: -1 *****
+// 30. 'engineering'.indexOf('neer') Prediction: undefined Result: 4 *****
+// 31. 'engineering'.indexOf('r') > -1 Prediction: true Result: true
+// 32. 'science'.indexOf('x') === -1 Prediction: false Result: true ****
 
 // List 3 expressions that surprised you or gave unexpected results, why?
+//1. 'engineering'.indexOf('neer')- I did not expect the results of this one because I thought that as the method searched the string for 'neer' it would fail to output an index value as I thought 'neer' would have a range of indexes. Thinking over it I realized that indexOf probably works slightly differently for a string and would just output the first index it finds for the string.
+//2. 'true' === true- This surprised me because I thought that since 'true' is a populated string, it would evaluate as truthy and then compare that to the boolean true and output as true.
+//3. 5 === 5.0  I thought that this would output as false because I thought JS would see that '5' and '5.0' are different, but I now see that since JS is a loose language, that since it evaluates that 5.0 is the same is 5, the result is true.
 
 // List 2 exprsssions you felt comfortable interpreting at a glance.
-
+//1. 'engineering'[3].toUpperCase()
+//2. !true && !(false || false)
 // List 1 expression you are still 'shaky' on, why?
-
+//'engineering'.indexOf('G')- I am shaky on this one because I am having a hard time understanding why a character that doesn't exist in the string has an index value instead of undefined. I understand that an index of -1 does not exist in the grand scheme of things, but I am not sure of the logic as to why that is the result of the evaluation and not undefined.
```

Comment:
So in JS the lowest index is 0 so returning `-1` won't cause confusion as no index can be `-1`